### DISCUSSION TITLE
[Question]How to reduce your memory footprint

### DISCUSSION THREAD

**Discussion by AthenaRoland** [flet-dev/flet#2627] [Q&A]
### Question

I created a web image viewing application, but loading the image caused the memory usage to be too high and stuttering
![C1B556818A3E3547909D89C67756F010](https://github.com/flet-dev/flet/assets/143491047/849a26aa-1200-4124-81fa-c4548b240e9c)
![QQ截图20240214171637](https://github.com/flet-dev/flet/assets/143491047/7f4e0132-291b-41d1-8a11-a9eac603a531)

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by 09u2h4n** [comment]
Maybe these can work;

### Image format:
* Some image formats (like .png) can be uncompressed and large. You should use jpeg or webp formats (Compressed)

### Number of images loaded:
* I think the program is loading multiple images at once this can be lead to high memory usage. Try load image when user scrolls them

### Caching:
* I am not sure how to implement to flet program but can be helpful
